plugins/generic/usageStats/locale/uk_UA/locale.xml

```diff
@@ -0,0 +1,30 @@
+<?xml version="1.0" encoding="UTF-8"?>

```

Comment:
The usageStats plugin has been removed from the master branch recently. Please remove the file from the PR and open another PR for the usageStats repository, if you want to.